examples/undo-tree/index.jsx

```diff
@@ -0,0 +1,53 @@
+import Microcosm from 'Microcosm'
+import React from 'react'
+import Drawing from './components/drawing'
+
+class UndoTree extends Microcosm {
+
+  shouldHistoryKeep(transaction) {
+    return true
+  }
```

Comment:
Really, I continue to struggle for a name here. It's "should the app commit this transaction into base state and purge it from history"